docs/projects/tcet-linux-website/frontend-components/installation-steps.mdx

```diff
@@ -0,0 +1,148 @@
+
+**Here's how we implemented code of this `Element` :**
+
+## Function Defination
+
+1. Opening the [**"InstallationSteps.jsx"**](https://github.com/tcet-opensource/tcet-linux-website/blob/main/src/components/InstallationSteps.jsx)file.
+
+2. In **Installation Steps Element**, We have implemented a `code` that consist a `React` component called **InstallationSteps**. It is a functional component that renders a set of installation steps for **TCET Linux Installation** along with `Corresponding Images` with **Hovering Effect**.
+
+  ``` jsx title="InstallationSteps.jsx" {1-2,11-12} showLineNumbers
+import React, { useState } from "react";
+    import welcomeImage from "../assets/image 9.png";
+    import locationImage from "../assets/image 10.png";
+    import keyboardImage from "../assets/image 11.png";
+    import storageImage from "../assets/image 12.png";
+    import userImage from "../assets/image 13.png";
+    import summaryImage from "../assets/image 14.png";
+    import installationImage from "../assets/image 15.png";
+    import finishImage from "../assets/image 16.png";
+function InstallationSteps() 
+    const [imageFirst, setImageFirst] = useState(welcomeImage);
+    const [imageSecond, setImageSecond] = useState(welcomeImage);
+    //Futher Code
+  ```
+3. Our implemented `code` begins with importing the necessary dependencies. `React`and the **UseState Hook** are imported from the React library. We have also imports several image files using relative paths for our `Requirements`.
+
+4. We have defined **InstallationSteps** function as the component.
+
+   - Inside the component, two **State Variables** are initialized using the `useState` hook: **imageFirst** and **imageSecond**.
+
+   - The **imageFirst** `state` variable represents the image displayed on the `Left side`, and **imageSecond** represents the image displayed on the `Right` side.
+
+## Array Defination 
+
+1. In this `step`, we have defined an Two `Array` called **installationStepsFirstSet** and **installationStepsSecondSet**. These arrays contain objects representing the installation steps.
+
+2. First **Array** contain the objects for `First Set` of Installation Steps component. Similarly Second **Array** contain the objects for `Second Set` of Installation Steps component.
+
+ ```jsx title="InstallationSteps.jsx" {2,4-8} showLineNumbers
+   //function defination
+   const installationStepsFirstSet = [
+    {
+      id: 1,
+      header: "Welcome Screen",
+      description:
+        "Launch the installer by searching for 'Install TCET Linux'. The installer will look like this.",
+      image: welcomeImage,
+    },
+    //Futher code
+ ```
+
+3. Each step object has properties such as id, header, description, and image which represents step number, step title, step description, step image file path.
+
+4. Similar Steps were carried out by us to create different object in both the **Arrays**.
```

Comment:
This should not come in a point and should be bold and also anchor tags must be added here too! to redirect it to #Array Definiton